package-lock.json

```diff
@@ -9994,11 +9994,6 @@
         "shallow-equals": "1.0.0"
       },
       "dependencies": {
-        "@react-native-community/netinfo": {
-          "version": "4.7.0",
-          "resolved": "https://registry.npmjs.org/@react-native-community/netinfo/-/netinfo-4.7.0.tgz",
-          "integrity": "sha512-a/sDB+AsLEUNmhAUlAaTYeXKyQdFGBUfatqKkX5jluBo2CB3OAuTHfm7rSjcaLB9EmG5iSq3fOTpync2E7EYTA=="
-        },
```

Comment:
How did this get impacted by the change?

Reply:
cause it was removed from one of the dependencies from mm-redux